reco_utils/docker/pyspark/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is great, we should have the images in dockerhub, maybe even we can have the official release and a dev release :-)

Reply:
Yea, the only thing you have to be mindful of is that we have to be careful of what we are allowed to redistribute. You may have to talk with CELA :(